src/gol.test.js

```diff
@@ -59,4 +25,94 @@ describe('countLiveNeighbours', () => {
     const expectedOutput = 3;
     expect(countLiveNeighbours(input, 3, 2)).toBe(expectedOutput);
   });
+  test('return 5 for border cell', () => {
+    const input = [
+      [0, 1, 1],
+      [1, 1, 1],
+      [1, 1, 1],
+    ];
+    const expectedOutput = 5;
+    expect(countLiveNeighbours(input, 1, 2)).toBe(expectedOutput);
+  });
+});
+
+describe('next', () => {
+  test('next return array of array', () => {
+    const input = [[]];
+    const output = next(input);
+    expect(Array.isArray(output)).toBe(true);
+    expect(Array.isArray(output[0])).toBe(true);
+  });
+  test('Rule 1: fewer than two live neighbours -> dies', () => {
```

Comment:
I like the structure of these tests and the split per business rule. It is interesting that you optimise the conditions in code but you keep the original business conditions in the tests. I like it!